tests/HypothesisClient/Exception/ApiExceptionTest.php

```diff
@@ -18,13 +17,6 @@ class ApiExceptionTest extends PHPUnit_Framework_TestCase
      */
     public function it_requires_a_message()
     {
-        try {
-            $this->getMockBuilder(ApiException::class)->getMock();
-            $this->fail('A message is required');
-        } catch (TypeError $error) {
-            $this->assertTrue(true, 'A message is required');
-            $this->assertContains('must be of the type string', $error->getMessage());
-        }
```

Comment:
These seem to be testing a type hint, which wouldn't need to go through a mock anyway.